code/_onclick/hud/alert.dm

```diff
@@ -40,7 +40,8 @@
 		var/old_plane = new_master.plane
 		new_master.layer = FLOAT_LAYER
 		new_master.plane = FLOAT_PLANE
-		alert.overlays += new_master
+		var/atom/yes_this_is_a_bloody_atom = alert
+		yes_this_is_a_bloody_atom.add_overlay(new_master)				// B Y O N D
```

Comment:
I realize that, but... calling `alert.add_overlay` won't compile for some reason because BYOND